src/main/java/uk/gov/hmcts/ccd/domain/service/common/SecurityClassificationService.java

```diff
@@ -55,26 +53,27 @@ public SecurityClassificationService(@Qualifier(CachedUserRepository.QUALIFIER)
                              .max(comparingInt(SecurityClassification::getRank));
     }
 
-    public Optional<CaseDetails> apply(CaseDetails caseDetails) {
+    public Optional<CaseDetails> applyClassification(CaseDetails caseDetails) {
         Optional<SecurityClassification> userClassificationOpt = getUserClassification(caseDetails.getJurisdiction());
         Optional<CaseDetails> result = Optional.of(caseDetails);
 
-        return result.filter(caseIfHasClassificationEqualOrLowerThan(userClassificationOpt))
```

Comment:
not sure why this change but if it's equivalent to the old version I think the old one is simpler to understand. 
Doing a filter first followed by a map is simpler than nesting the filtering inside and doing a flat map